src/test/scala/scalaz/nio/SelectorSuite.scala

```diff
@@ -66,9 +66,9 @@ object SelectorSuite extends DefaultRuntime {
           channel  <- ServerSocketChannel.open
           _        <- channel.bind(address)
           _        <- channel.configureBlocking(false)
-          key      <- channel.register(selector, JSelectionKey.OP_ACCEPT)
-          buffer   <- Buffer.byte(256)
-          _        <- started.succeed(())
+//          key      <- channel.register(selector, JSelectionKey.OP_ACCEPT)
```

Comment:
fmt linter got failed on unused ones. so i commented them for now

Reply:
@ratoshniuk this line is needed for the test to pass. Just change `key` to `_` and it should work.

Reply:
@ratoshniuk can you address this comment?